bin/bb8/targets.py

```diff
@@ -32,6 +34,15 @@ def mount_id(self):
     def before_restore(self):
         pass
 
+    def files_exist_locally(self):
+        test_empty_cmd = '([ -z "$(ls -A /data)" ] && echo "Empty") || echo "NotEmpty"'
+        volumes = {self.mount_id: {"bind": "/data", "mode": "ro"}}
+        output = self.docker.containers.run("bash",
+                                            command=["bash", "-c", test_empty_cmd],
+                                            volumes=volumes,
+                                            remove=True)
+        return output.decode('utf-8').strip() != "Empty"
```

Comment:
This seems a reasonable approach to me, but would be good to get @richfitz's input on bash.

Reply:
My medium-term thoughts here are are that we might be better off defining a new container (bb8-utils perhaps) that builds off of the bash one and include utilities; we're doing similar logic in `last_modified_local` (https://github.com/vimc/bb8/blob/4f68c9ac21d5e6e891bf0b3a635f0da4fd6b754e/bin/bb8/status.py#L52-L60) and moving to a small set of scripts would be nice

It *should* be probably simpler to detect the exit code from docker rather than the string that is returned, but the python docker package makes you jump through the hoops to get the status code from `run` (it's easy enough from the lower-level API but then you have trouble auto-cleaning containers)